app/static/src/app/components/options/ParameterSetView.vue

```diff
@@ -110,33 +131,57 @@ export default defineComponent({
     setup(props) {
         const store = useStore();
         const currentParams = computed(() => store.state.run.parameterValues);
+        const parameterColors = {
+            red: "#dc3545",
+            blue: "#479fb6",
+            grey: "#bbb"
+        };
+        const parametersToShow = computed(() => (store.state.run.showUnchangedParameters
+            ? props.parameterSet.parameterValues
+            : Object.fromEntries(
+                Object.entries(props.parameterSet.parameterValues).filter(
+                    ([name, value]) => value !== currentParams.value[name]
```

Comment:
good catch updates... Yeah probs should be fixed right --> easiest  way may be to just delete all saved parameter sets when any parameters are added / removed as the saved set is invalid now

Reply:
There is already a ticket to deal with model parameters changing under the parameter sets:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3910/Update-parameter-sets-when-parameters-in-model-change

Correct behaviour still to be defined I think. We've got away with this because I think the defined parameters are generally pretty stable before people start playing with parameter sets, but it's definitely an anomaly we should address at some point.